src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      class="container"
```

Comment:
```suggestion
      class="container large"
```
Давай в нас буде один загальний клас, який міститиме загальні стилі (наприклад, container) і інші класи, які будуть містити стилі конкретно для великих чи малих елементів, наприклад:
```
container large
container small
icon large
icon small
search large
search small
```